GVFS/GVFS.Service/GVFSService.Mac.cs

```diff
@@ -16,13 +20,32 @@ public class GVFSService
         private Thread serviceThread;
         private ManualResetEvent serviceStopped;
         private string serviceName;
+        private RepoRegistry repoRegistry;
+        private string serviceDataLocation;
+        private RequestHandler requestHandler;
 
         public GVFSService(JsonTracer tracer)
         {
-            this.tracer = tracer;
+            if (tracer != null)
```

Comment:
Why would the `tracer` ever be `null`? As best I can tell we don't have this check in the Windows code.